shared/src/main/scala/squants/package.scala

```diff
@@ -118,4 +119,13 @@ package object squants {
     def /(that: Time) = Each(bd) / that
     def per(that: Time): Frequency = /(that)
   }
+
+  /**
+    * Provides an implicit conversion from Dimensionless to Double, allowing a Dimensionless value
+    * to be used anywhere a Double (or similar primitive) is required
+    *
+    * @param d Dimensionless
+    * @return
+    */
+  implicit def DimensionlessToDouble(d: Dimensionless): Double = d.toEach
```

Comment:
To support the feature, we really need to support this conversion.  Not sure how an implicit class would work as no method is being called on the Dimensionless.  It's just being used in an op.  If we don't want to do this, the answer is to force the user to call toEach on the Dimensionless - which is effectively a toDouble (maybe we want to add that as an alias) - in places where Double is required.  In the use case described in the issue we end up with:

```
val net = EUR(100)
val tax = 22.percent
val totalIncludingTax: Money = net + net * tax.toEach (or toDouble)
```

To avoid that explicit call, I think we will need the implicit conversion.

Perhaps having it the package is not a great idea as it will get pulled into scope to readily.  There other implicit conversions like this, for example, timeToScalaDuration and the scalaDurationtoTime methods.  However, they are defined TimeConversion object.

Perhaps it would be best to move this to DimensionalessConversion object.